packages/vsce/README.md

```diff
@@ -124,71 +124,71 @@ Right-click a CICS profile and select `Manage Profile` to show profile options.
+Expand a CICS region to show folders for the resource types `Programs`, `Transactions`, `Local Files`, and `Tasks` with the set of resources nested within each. The number of resources in a resource tree appear in square brackets next to the tree name.
 
 The list of resources are pre-filtered to exclude many of the IBM-supplied items; these default filters are configured in the settings for the CICS extension.
 
-Use the search icon <img src="/packages/vsce/docs/images/resource-filter.png" width="16px"/> against a resource type to apply a filter. This can be an exact resource name or can include wildcards. The search history is saved so you can recall previous searches.
+Use the **Search** icon <img src="/packages/vsce/docs/images/resource-filter.png" width="16px"/> against a resource type to apply a filter. This can be an exact resource name or can include wildcards. The search history is saved so you can recall previous searches.
 
-To reset the filter to its initial criteria use the clear filter icon <img src="/packages/vsce/docs/images/resource-filter-clear.png" width="16px"/> against the resource type. If you wish to see all resources in a region (including IBM supplied ones) you can use "\*" as a filter.
+To reset the filter to its initial criteria, use the **Clear Filter** icon <img src="/packages/vsce/docs/images/resource-filter-clear.png" width="16px"/> against the resource type. If you want to see all resources in a region (including IBM supplied ones) you can use `\*` as a filter.
 
 <p align="center">
 <img src="/packages/vsce/docs/images/region-filter.gif" alt="Zowe CICS Explorer Filter" width="700px"/>
 </p>
 
-**Tip:** To apply multiple filters, separate entries with a comma. You can append any filter with an \*, which indicates wildcard searching.
+**Tip:** To apply multiple filters, separate entries with a comma. You can append any filter with an `\*`, which indicates wildcard searching.
 
-### Show and Filter Resources in a Plex
+### Show and filter resources in a plex
 
-In the same way as filtering resources in a region, you can apply a filter on resources in a plex. Use the filter and clear filter buttons against the Regions tree element to manage filters across the whole set of regions and resources.
+In the same way as filtering resources in a region, you can apply a filter on resources in a plex. Use the **Filter** and **Clear Filter** buttons against the Regions tree element to manage filters across the whole set of regions and resources. **[Should "filter" and "clear filter" be "search" and "clear search"? the icons looks like "search" icons rather than "filter" icons. From what I see, seems like we're using "search" and "filter" interchangeably and we should use only one of these terms.]**
```

Comment:
@davenice -- please respond

Reply:
This is a good point. I agree that the icon is a bit 'searchy' but we are filtering. I think this was a lack of direction with some of the previous iconography and wording.

I've fixed the text to talk about filtering instead of searching - @Rajat-Sahrawat can consider whether we can switch the visual paradigm to filtering without confusing all of our users!